### DISCUSSION TITLE
Include Translated fields in quicksearch

### DISCUSSION THREAD

**Discussion by alexvdvalk** [directus/directus#22600] [Feature Requests]
### Summary

Currently when using the free text search on a collection, related collections are ignored. This makes sense, but in cases where translations are heavily used, it would be great to include the translation table in the search.

Additionally, it should only search translations in the user's selected language.

### Basic Example

tbd

### Motivation

In environments heavily using translations, the freetext search seems to not work correctly.

### Detailed Design

tbd

### Must Have:
- Freetext search on a collection with the ability to traverse relationships and return items based on fields in related collections
- 

### Should Have:
- Toggle which related collections are included in a free text search

### Could Have:

- Ability to specify a base language for free text searches. for example, to always include the english translations in the freetext search
- User specified related tables to include in the freetext search, with filters with variable support ($CURRENT_USER)

### Won't Have:
- Results highlighting

### Drawbacks

Performance degradation as searches will have to include additional tables.

There may be cases where you use the free text search and unexpected results come back

### Alternatives

Users can view the translations table directly, but this wouldnt work in many cases, inside a relational drawer for example.

Users can explicitly search the Translations > [field name] field but this is many more clicks

### Adoption Strategy

There should be no breaking changes, but some documentation may need to be updated to explain how items are found when using the search

### Unresolved Questions

How many levels of relationships should be included in the freetext search? Should this be configurable?

**Comment by JoshTheDerf** [comment]
See also:
- https://github.com/directus/directus/discussions/10517
- https://github.com/directus/directus/discussions/12250
- https://github.com/directus/directus/discussions/21720

**Comment by JoshTheDerf** [comment]
We built an extension to accomplish this:
https://github.com/cmiteam/directus-extension-custom-search
